src/router/Routes.tsx

```diff
@@ -13,13 +13,15 @@ import TopBar from "../TopBar/TopBar";
 import SideBar from "../SideBar/SideBar";
 import SignUp from "../signUp/SignUp";
 import "./Routes.css";
+import { ToastContainer, toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import Book from "../Book/Book";
 import BookScreen from "../Book/BookScreen";
 import Profile from "../Profile/Profile";
 import "../common/Notify.css";
-import {isAdmin as isAdminMock} from "../utils/mocksettings.json"; //Once integration with login is ready, this sould be deleted
-
+import Login from "../Login/Login"
+import "../common/Notify.css"
+import { loggedUser, isAdmin } from "../utils/mocksettings.json";
```

Comment:
[NOT ACTION REQUIRED] When the onStart feature is merged, this mocked admin should be deleted.

Reply:
Now the onStart feature is merged on sprint_1.